playbooks/update-provider-admin-password.yml

```diff
@@ -2,10 +2,10 @@
 - hosts: controller[0]
   gather_facts: no
   tasks:
-  - name: update provider_admin password in keystone
-    shell: . /root/stackrc && keystone user-list | grep provider_admin | awk '{print $2}'
+  - name: update admin password in keystone
+    shell: . /root/stackrc && keystone user-list | grep admin | awk '{print $2}'
     register: keystone_uuid
-  - shell: . /root/stackrc && keystone user-password-update --pass {{ secrets.provider_admin_password }} {{ keystone_uuid.stdout }}
+  - shell: . /root/stackrc && keystone user-password-update --pass {{ secrets.admin_password }} {{ keystone_uuid.stdout }}
```

Comment:
This would get executed if we decided to rotate admin passwords. The password is saved in each cloud's environment file, we'd change the password in the file then run this automation to change it in the cloud. We'll want to rotate the passwords periodically.